packages/employer-api/src/config/config.ts

```diff
@@ -5,6 +5,7 @@ const formsBaseUrl = process.env.FORMS_API_URL || ""
 const cdogsBaseUrl = process.env.CDOGS_HOST || ""
 const authBaseUrl = process.env.AUTH_KEYCLOAK_SERVER_URL || ""
 const oesBaseUrl = process.env.OES_URL || ""
+const fromAPI = process.env.FORM_API_URL || ""
```

Comment:
Sorry my previous comment was ambiguous! I meant fix typo 'fromAPI' to 'formAPI'.

Reply:
Fixed typo